curricula/urls.py

```diff
@@ -9,6 +9,7 @@
 
                        # Allow viewing concept documentations without the /documentation/ prefix
                        # so links work on both curriculumbuilder and docs.code.org
+                       url(r'^(?P<slug>/concepts.*)/$', documentation_views.map_view, name="map_view"),
```

Comment:
This adds in the old code that used to be here. The line below was changed in the process of trying to figure out how to make these links but never got reverted.

Reply:
I thought having both would be good in hopes that it would catch old urls that pointed to this page but hopefully the /docs/concepts route will work to move everything over to using the same url format